core/src/error.rs

```diff
@@ -21,7 +21,7 @@ impl From<CrypError> for Error {
 
 #[derive(Debug, PartialEq, Eq)]
 pub enum TxError {
-    OutofBound,
+    OutOfBound,
```

Comment:
Can we rename this `OutOfBound` error to something else? I think it's only used now when capacity doesn't match. But it happens to collide with an error with the same name at VM side: https://github.com/nervosnetwork/ckb-vm/blob/79bc6e33a42902c54a5479c96efbfb8898bb7cdc/src/lib.rs#L72 I suggest we rename one of them to avoid confusion in logs.

Reply:
Agreed, `OutOfBound` is ambiguous to describe exceed capacity error.